e2e/requirements.txt

```diff
@@ -0,0 +1,5 @@
+kubernetes
```

Comment:
Python packages tend to break stuff between the versions, maybe it makes sense think about using fixed versions.